nuxt.config.ts

```diff
@@ -71,7 +71,11 @@ export default defineNuxtConfig({
     async (_options, nuxt) => {
       nuxt.hooks.hook('vite:extendConfig', config =>
         // @ts-expect-error: Do we really need to extend the config ?
-        config.plugins.push(vuetify()))
+        config.plugins.push(vuetify({
+          autoImport: {
+            labs: true,
+          },
+        })))
```

Comment:
Finnaly we need this ?

Reply:
yes to be able to use VDataTable as it's a Lab Component

Reply:
or we need to upgrade Vuetify to v.3.4.0 at least (cf: https://github.com/vuetifyjs/vuetify/commit/6c3f8335a8d96c7f9fe5418ce2684ce04582f5cb)

Reply:
In `plugins/vuetify.ts`
```
import { createVuetify } from 'vuetify'
import { aliases, mdi } from 'vuetify/iconsets/mdi-svg'
import { VDataTable } from 'vuetify/labs/VDataTable'
import { VSkeletonLoader } from 'vuetify/labs/VSkeletonLoader'
import 'vuetify/styles'

import { defineNuxtPlugin } from '#app/nuxt'

export default defineNuxtPlugin((nuxtApp) => {
  const vuetify = createVuetify({
    components: {
      VDataTable,
      VSkeletonLoader,
    },
    icons: {
      defaultSet: 'mdi',
      aliases,
      sets: {
        mdi,
      },
    },
    ssr: true,
  })

  nuxtApp.vueApp.use(vuetify)
})
```